.circleci/config.yml

```diff
@@ -214,7 +215,7 @@ workflows:
             - tier-1-tap-user
           requires:
             - 'Testing bookmarks'
-build_daily:
+  build_daily:
```

Comment:
Fixed the build_daily workflow definition to run the tests once a day. ([Reference](https://github.com/singer-io/tap-stripe/pull/116))